examples/package.json

```diff
@@ -7,5 +7,8 @@
   "dependencies": {
     "scylladb-nodejs-rs-driver": "file:./../",
     "async": "^1.5.2"
+  },
+  "scripts": {
+    "examples": "node ./runner.js"
```

Comment:
:pick: Please make this a separate commit, as it introduces new functionality not connected to the README itself.